src/discussions/common/AuthorLabel.jsx

```diff
@@ -46,7 +46,6 @@ function AuthorLabel({
     <div className={className}>
       <span
         className={classNames('mr-1 font-size-14 font-style-normal font-family-inter font-weight-500', {
-          'text-primary-500': !authorLabelMessage && !isRetiredUser,
```

Comment:
We are using this component in the post-summary list too. We need this color there. If you do not need this color than you can add some conditions here.